### PR TITLE
[Bug Fix] LPS-137136 Jdom | resend

### PR DESCRIPTION
**Pull request by dantewang**
@tinatian This patch always sets false to the parser being built.

### CODE REVIEW

**Comment by dantewang** [comment]
ci:test:sf

**Comment by dantewang** [comment]
ci:test:relevant

**Comment by dantewang** [comment]
ci:test

**Comment by zhangyan0701** [comment]
No related failures.

The failed tests were because the mysql docker on Jenkins didn't startup.

**Comment by tinatian** [comment]
@dantewang Is the patch enough? I saw there is still usage of variable "expand"

Tina

File: portal-impl/src/portal.properties [hunk 1]
```diff
@@ -8362,7 +8361,6 @@
         WEB-INF/lib/portal-kernel.jar,\
         WEB-INF/lib/portlet.jar,\
         WEB-INF/lib/portletmvc4spring-framework.jar,\
-        WEB-INF/lib/rome.jar,\
```
Comment (tinatian) [inline portal-impl/src/portal.properties:8365]: @dantewang Is it safe to remove without adding the new ones?

File: .classpath [hunk 2]
```diff
@@ -263,7 +263,6 @@
 	<classpathentry kind="lib" path="lib/portal/rhino.jar"/>
 	<classpathentry kind="lib" path="lib/portal/ridl.jar"/>
 	<classpathentry kind="lib" path="lib/portal/rmi-api.jar"/>
-	<classpathentry kind="lib" path="lib/portal/rome.jar"/>
```
Comment (tinatian) [inline .classpath:266]: @dantewang Same thing here, is it safe to remove wihout adding new ones?

File: portal-impl/src/portal.properties (hunk 1 above)
Comment (dantewang) [reply portal-impl/src/portal.properties:8365]: @tinatian rome's replacement rometools is already in portal.

**Comment by dantewang** [comment]
@tinatian I will resend and remove the `expand` field completely.

**Pull request closed without merging**